war/src/test/java/executable/MainTest.java

```diff
@@ -0,0 +1,84 @@
+package executable;
+
+import edu.umd.cs.findbugs.annotations.CheckForNull;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+class MainTest {
```

Comment:
Copied from https://github.com/jenkinsci/extras-executable-war/blob/b04de7c7efde96d5a31c11a81611c8679b6eb128/src/test/java/MainTest.java and adapted for JUnit 5.

Reply:
This introduces a horrendous amount of logspam during the build.
When looking for the reason that builds fail this test output is the first that you see and it is confusing as it is test output but is not redirected to a file.

Reply:
That log spam already existed when running the tests in `extras-executable-war`, it was just moved to core. The root cause of the problem is that the core parent POM lacks any settings to cut down on log spam like the plugin parent POM does: https://github.com/jenkinsci/pom/pull/273

Reply:
@jtnord This was a pre-existing issue in `extras-executable-war` where this test originally lived. In the future, kindly refrain from commenting on my PRs about pre-existing issues.